### ISSUE TITLE
Feather loses usb/serial connection on Windows

### ISSUE THREAD

**Issue by tamberg** [tamberg/mse-tsm-mobcom#4]
https://forums.adafruit.com/viewtopic.php?f=57&t=170888

**Comment by danielhass** [comment]
This issue was caused by the somewhat confusing naming of the Sense Feather board. I chose the wrong board type during the flash process `Adafruit Bluefruit nRF52840 Feather Express` which is only partly HW-compatible to the Bluefruit Sense Board.
Most of the sensor and actuators work fine but the BLE module is causing problems with the wrong board type selected. The following call in the Bluefruit library never returns: https://github.com/adafruit/Adafruit_nRF52_Arduino/blob/master/libraries/Bluefruit52Lib/src/bluefruit.cpp#L319 - presumably due to a messed up clock setting as result of the wrong board type selection. This causes the whole board to freeze as soon as `Bluefruit.begin();` is called.

Additions for the troubleshooting guide:

- If the Board freezes / gets unresponsive after the Bluefruit.begin() call:
   - press reset twice fast to exit frozen state and enter bootloader mode
   - make sure to select the correct board type in the Arduino IDE: Tools > Board > Adafruit Feather Bluefruit Sense

@tamberg - this can be closed from my point of view, sorry for the inconvenience caused...
